README.md

```diff
@@ -10,7 +10,7 @@ For more details and examples of tools that use this project, you can check
 Add to your SBT dependencies:
 
 ```
-"com.codacy" %% "codacy-metrics-scala-seed" % "1.0.0-SNAPSHOT"
```

Comment:
Missing the version badge. Check `https://github.com/codacy/codacy-coverage-reporter`